cdap-docs/tools/cdap-default/cdap-default-deprecated.xml

```diff
@@ -23,15 +23,15 @@
     <name>app.bind.address</name>
     <value>0.0.0.0</value>
     <description>
-      App Fabric service bind address (deprecated: use master.services.bind.address instead)
+      App Fabric service bind address (deprecated: replaced with ${master.services.bind.address})
```

Comment:
@awholegunch the convention in code (Java files) is to use the phrase ``@deprecated since x.y.z. use {@link foo} instead.`` in javadocs. Should we keep it consistent?